### DISCUSSION TITLE
初次启动生成配置文件

### DISCUSSION THREAD

**Discussion by tidexe** [Jinnrry/PMail#148] [Ideas]
初次启动没有生成配置文件，我无法得知具体的配置文件如何配置。
再说，启动后不是会有web初次配置吗，不巧的是我的服务器在国内，http协议必封，所以无法在配置完成前配置ssl。
但是，我计划使用内网穿透到国外在进行配置。

**Comment by tidexe** [comment]
已解决，在config/config.json

**Reply by tidexe** [reply]
但是我觉得还是要写一写配置文件的文档

**Reply by Jinnrry** [reply]
Readme里面解释了每一行配置是干啥用的
